extensions/amp-story-auto-ads/0.1/story-ad-ui.js

```diff
@@ -182,6 +192,63 @@ export function handleAttributionClick(win, href) {
   openWindowDialog(win, href, '_blank');
 }
 
+/**  Creates a page-outlink element, returns an anchor tag containing relevant data if successful
+ * @param {!Document} doc
+ * @param {!StoryAdUIMetadata} uiMetadata
+ * @param {!Element} container
+ * @return {?Element}
+ */
+function createPageOutlink_(doc, uiMetadata, container) {
+  const pageOutlink = doc.createElement('amp-story-page-outlink');
+  pageOutlink.setAttribute('layout', 'nodisplay');
+
+  const pageAnchorTag = doc.createElement('a');
```

Comment:
can we reuse the anchor created here? https://github.com/ampproject/amphtml/blob/b19d29643a29bbb57f3230babb3c72ba99d693ee/extensions/amp-story-auto-ads/0.1/story-ad-ui.js#L196

Right now if the experiment is on we create that element and drop it. I think we can reuse it if we get rid of its styling and let amp-story-page-outlink style it.

This also made me realize we use the old UI to measure if the text is going to fit. I don't think we need to fix this now but please add todo to clean up measurement when we launch new UI.

Reply:
Yes, there will be a more precise code cleanup when we refactor and remove the other element. 
I will do a thorough sweep.